docs/statistics.md

```diff
@@ -0,0 +1,292 @@
+SRT provides a powerful set of statistical data on a socket. This data can be used to keep an eye on socket's health, and track some faulty situations as well.
+
+Statistics is calculated on each side (receiver and sender) independently, and is not exchanged between peers, unless explicitly stated.
```

Comment:
Statistics are calculated independently on each side (receiver and sender), and are not exchanged between peers, unless